### ISSUE TITLE
synctex doesn't handle \input without .tex extension

### ISSUE THREAD

**Issue by znewman01** [tectonic-typesetting/tectonic#744]
If I `\input{foo}` (rather than `\input{foo.tex}`), `tectonic --synctex` generates a file with `foo` as the filename rather than `foo.tex`. This breaks the synctex-compatible viewers I've tried.

```shell
$ cat test.tex
\documentclass{article}
\begin{document}
\input{test-input}
\end{document}
$ cat test-input.tex
Hello world!
$ pdflatex -synctex=1 test.tex > /dev/null; gunzip -c test.synctex.gz  | grep test-input
Input:6:/home/zjn/git/spectrum-paper/./test-input.tex
$ tectonic --synctex test.tex > /dev/null; gunzip -c test.synctex.gz | grep input
Input:5:test-input
```

Thanks for Tectonic; I use it for every project I can!

**Comment by pkgw** [comment]
Thanks for reporting. This is probably straightforward to fix, and should be tackled jointly with #720.

We're always looking to recruit new contributors to the project so if you or someone you know would like to try submitting a PR to fix this bug, please comment here and we'll help as best we can!

**Comment by znewman01** [comment]
Seems to work in Emacs:
![recording-20210609-152745](https://user-images.githubusercontent.com/873857/121416908-587e7980-c937-11eb-9518-39dc07998755.gif)
